oberoncms/adapter-pgsql/scripts/prepare.mjs

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env node
+/* eslint-env node */
+import dotenv from "dotenv"
+dotenv.config({ path: ".env.local" })
+
+import { createWriteStream } from "fs"
+import { mkdir } from "fs/promises"
+import { migrate } from "drizzle-orm/node-postgres/migrator"
+
+import { drizzle } from "drizzle-orm/node-postgres"
+import { pgTable, text } from "drizzle-orm/pg-core"
+
+import Pg from "pg"
+
+const { Pool } = Pg
+
+const pages = pgTable("pages", {
+  key: text("key").notNull().primaryKey(),
+  data: text("data"),
+})
+
+const createRemoteClient = async () => {
+  if (!process.env.DATABASE_URL) {
+    throw new Error(
+      "No remote database credentials supplied: have you set database credentials?",
+    )
+  }
+  return new Pool({
+    connectionString: process.env.DATABASE_URL,
+  })
+}
+
+const createLocalClient = async () => {
```

Comment:
I don't think you need a remote and local. The only reason they're separate in the turso is adapter is it uses a different library for local dev.